Mathlib/CategoryTheory/Opposites.lean

```diff
@@ -250,6 +250,9 @@ protected def rightOp (F : Cᵒᵖ ⥤ D) : C ⥤ Dᵒᵖ where
   map f := (F.map f.op).op
 #align category_theory.functor.right_op CategoryTheory.Functor.rightOp
 
+lemma rightOp_map_unop {F : Cᵒᵖ ⥤ D} {X Y} (f : X ⟶ Y) :
```

Comment:
```suggestion
@[simp]
lemma rightOp_map_unop {F : Cᵒᵖ ⥤ D} {X Y} (f : X ⟶ Y) :
```

Reply:
I think this is already provable by simp (`Functor.rightOp_map` + `Quiver.Hom.unop_op`)

Reply:
indeed